packages/payload-cloud/src/plugin.ts

```diff
@@ -93,5 +95,79 @@ export const payloadCloudPlugin =
       })
     }
 
+    // We set up cron jobs on init.
+    // We also make sure to only run on one instance using a instance identifier stored in a global.
+
+    // If you modify this defaults, make sure to update the TsDoc in the types file.
+    const DEFAULT_CRON = '* * * * *'
+    const DEFAULT_LIMIT = 10
+    const DEFAULT_CRON_JOB = {
+      cron: DEFAULT_CRON,
+      limit: DEFAULT_LIMIT,
+      queue: 'default (every minute)',
+    }
+    config.globals = [
+      ...(config.globals || []),
+      {
+        slug: 'payload-cloud-instance',
+        admin: {
+          hidden: true,
+        },
+        fields: [
+          {
+            name: 'instance',
+            type: 'text',
+            required: true,
+          },
+        ],
+      },
+    ]
+    const newOnInit = async (payload) => {
+      if (config.onInit) {
+        await config.onInit(payload)
+      }
+      const instance = generateRandomString()
+
+      await payload.updateGlobal({
+        slug: 'payload-cloud-instance',
+        data: {
+          instance,
+        },
+      })
+
+      await waitRandomTime()
+
+      const cloudInstance = await payload.findGlobal({
+        slug: 'payload-cloud-instance',
+      })
+
+      const cronJobs = pluginOptions?.cronJobs?.run ?? [DEFAULT_CRON_JOB]
+      if (cloudInstance.instance === instance) {
+        cronJobs.forEach((cronConfig) => {
+          new Cron(cronConfig.cron ?? DEFAULT_CRON, async () => {
+            await payload.jobs.run({
+              limit: cronConfig.limit ?? DEFAULT_LIMIT,
+              queue: cronConfig.queue,
+            })
+          })
+        })
+      }
+    }
+
+    config.onInit = newOnInit
+
     return config
   }
+
+function waitRandomTime(): Promise<void> {
+  const min = 1000 // 1 second in milliseconds
+  const max = 5000 // 5 seconds in milliseconds
```

Comment:
The window of time does not seem long enough to gaurentee that multiple instances aren't self identifying as the cron runner.
A min of 1 second between starting up the app instance, connecting to the database, and updating the global all adds up to a much larger tolerance needed if I had to guess.

I'm not sure how much you'ved tested this, but to me we should remove the  `waitRandomTime()` entirely and just let the instances write their unique id to the global as they all come up. Then in a `setTimeout` with 2 full minutes do lines 140 - 155.